rules/base-os-compatibility-version.mk

```diff
@@ -0,0 +1,6 @@
+# Define SONiC base OS compatibility version.
+# SONiC packages dependning on specific SONiC base image features (kernel change, etc.)
+# may define a dependency on specific version pattern.
+
+BASE_OS_COMPATIBILITY_VERSION = 1.0.0
```

Comment:
What is the plan behind  on update of this version ?
How is it mandated ?

W/o a plan in place, this will lose its value

For e.g. Image version is mandated, kernel version is picked.

Reply:
@yxieca, @lguohan, @ben-gale, @liat-grozovik  -- We could start with these folks to get some answer